### DISCUSSION TITLE
delete points from mesh

### DISCUSSION THREAD

**Discussion by ghazalbangash** [pyvista/pyvista#1616] [Q&A] [answered]
Hi there 
I want to delete/remove some points from my mesh and update it inplace ,any idea how to do it?
any help would be highly appreciated

**Comment by akaszynski** [comment, marked as answer]
Try [remove_points]( https://docs.pyvista.org/core/filters.html?highlight=remove_points#pyvista.PolyDataFilters.remove_points) with `inplace=True`
